obs-browser-plugin.cpp

```diff
@@ -567,7 +567,7 @@ static void check_hwaccel_support(void)
 bool obs_module_load(void)
 {
 	blog(LOG_INFO, "[obs-browser]: Version %s", OBS_BROWSER_VERSION_STRING);
-
+	blog(LOG_INFO, "[obs-browser]: Cef Version %s", CEF_VERSION);
```

Comment:
I'd capitalise it as CEF Version, but otherwise this works as expected.

Reply:
ok done